FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.2.1
+- [fixed]Check for Pending Dynamic link guard check logic
```

Comment:
Does this fix an existing GitHub issue? If so please add the number.

Casing fixes:
`[fixed] Check for pending dynamic Link guard check logic.`

Reply:
This is an addon fix to PR https://github.com/firebase/firebase-ios-sdk/pull/5665